### PR TITLE
more functionlity for MC events added

### PR DESCRIPTION
**Pull request by fibnumb**

### CODE REVIEW

File: PWGJE/EMCALJetTasks/macros/AddTaskEmcalJetSpectra8TeVTriggerQA.C [hunk 1]
```diff
@@ -2,6 +2,7 @@ AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA* AddTaskEmcalJetSpectra8TeVTriggerQA
                                                                                  const char *ntracks            = "usedefault",
                                                                                  const char *nclusters          = "usedefault",
                                                                                  const char* ncells             = "usedefault",
+                                                                                 const char *nparticle          = "mcparticles",
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/macros/AddTaskEmcalJetSpectra8TeVTriggerQA.C]: Did you want to use this somewhere in the AddTask?

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx [hunk 2]
```diff
@@ -531,6 +587,7 @@ Bool_t AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::FillHistograms()
     DoTrackLoop();
     DoClusterLoop();
     DoCellLoop();
+    //DoParticleLoop();
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: Did you intend to leave this commented out?

**Comment by raymondEhlers** [comment]
+test

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx (hunk 2 above)
Comment (fibnumb) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: yes,  Im still adding to this and it is crashing right now over data.  So I'll leave it out for now

Comment (raymondEhlers) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: Okay - the purpose of this commit isn't really clear to me then, but I'll just +1 it

**Comment by raymondEhlers** [comment]
+1

**Pull request merged**